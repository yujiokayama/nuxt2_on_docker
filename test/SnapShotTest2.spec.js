import { shallowMount } from '@vue/test-utils'
import SnapShotTest2 from '@/components/test/SnapShotTest2.vue'

describe('コンポーネントのテスト', () => {
  test('何もレンダリングされないケース', () => {
    const wrapper = shallowMount(SnapShotTest2)

    expect(wrapper.element).toMatchSnapshot()
  })

  test('文字列を渡すケース', () => {
    const wrapper = shallowMount(Component, {
      propsData: {
        message: 'Vue Test Utils',
      },
    })

    expect(wrapper.element).toMatchSnapshot()
  })
})
