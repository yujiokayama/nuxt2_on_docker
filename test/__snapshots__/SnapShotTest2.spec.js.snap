// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`コンポーネントのテスト 何もレンダリングされないケース 1`] = `
<p>
  
</p>
`;

exports[`コンポーネントのテスト 文字列を渡すケース 1`] = `
<p>
  Vue Test Utils
</p>
`;
